java/cdk/src/main/java/sleeper/cdk/stack/QueryQueueStack.java

```diff
@@ -74,7 +77,9 @@ private Queue setupQueryQueue(InstanceProperties instanceProperties) {
         instanceProperties.set(CdkDefinedInstanceProperty.QUERY_QUEUE_ARN, queryQueue.getQueueArn());
         instanceProperties.set(CdkDefinedInstanceProperty.QUERY_DLQ_URL, queryQueueForDLs.getQueueUrl());
         instanceProperties.set(CdkDefinedInstanceProperty.QUERY_DLQ_ARN, queryQueueForDLs.getQueueArn());
-
+        createAlarmForDlq(this, "QueriesAlarm",
+                "Alarms if there are any messages on the dead letter queue for the queries queue",
+                queryQueueForDLs, topic);
```

Comment:
Maybe we should handle these failures separate since they can be client errors. We can make that a separate issue if we want to do that though.